pkg/utils/hash.go

```diff
@@ -0,0 +1,27 @@
+package utils
+
+import (
+	"encoding/json"
+	"fmt"
+	"hash/fnv"
+
+	"helm.sh/helm/v3/pkg/chart"
+	"helm.sh/helm/v3/pkg/chartutil"
+)
+
+// Hash creates a 32-bit FNV-1 hash of the Helm chart including the values
+func Hash(chart *chart.Chart, values chartutil.Values) string {
+	algorithm := fnv.New64()
```

Comment:
can you add some tests here?
does the same chart give the same hash when executed 2 times (idempotent)?

Reply:
I added some tests and enhanced the hash function. The file name is now also part of the hash and the file order will be ignored.